ultralytics/nn/modules/block.py

```diff
@@ -238,6 +238,13 @@ def forward(self, x):
         y.extend(m(y[-1]) for m in self.m)
         return self.cv2(torch.cat(y, 1))
 
+    def forward_fx(self, x):
+        """Forward pass creating list explicitly instead of using list()."""
+        y1 = self.cv1(x).chunk(2, 1)
+        y = [y1[0], y1[1]]
+        y.extend(m(y[-1]) for m in self.m)
+        return self.cv2(torch.cat(y, 1))
```

Comment:
@ambitious-octopus @Laughing-q forward_split() below is used for export for all formats including ONNX. Don't you want to modify this method instead of forward()?

Reply:
@glenn-jocher make sense! did the update in https://github.com/ultralytics/ultralytics/pull/14878/commits/5fbcdb1bb0f9d689b8cc3cce102603b625f1f806